src/NHibernate/Mapping/ByCode/IModelInspector.cs

```diff
@@ -7,7 +7,7 @@ public interface IModelInspector
 	{
 		bool IsRootEntity(System.Type type);
 		bool IsComponent(System.Type type);
-		bool IsEntity(System.Type type);
+		bool IsEntity(System.Type type, bool explicitlyDeclared = false);
```

Comment:
Since the interface is changed, an IsDeclaredEntity method would make more sense, or is there something I'm not seeing?